solutions/swb-reference/SWBv2.openapi.json

```diff
@@ -0,0 +1,2541 @@
+            "example": ""
+          },
+          "description": "ID of environment type associated with this config"
+        }
+      ]
+    },
+    "/environmentTypes/{envTypeId}/configurations/{envTypeConfigId}": {
+      "get": {
+        "tags": ["envTypeConfig"],
+        "summary": "List projects by envTypeConfig",
+        "description": "List projects by envTypeConfig",
+        "operationId": "listProjectsByEnvtypeconfig",
+        "parameters": [
+          {
+            "name": "paginationToken",
+            "in": "query",
+            "schema": {
+              "type": "string",
+              "example": ""
+            }
+          },
+          {
+            "name": "pageSize",
+            "in": "query",
+            "schema": {
+              "type": "string",
+              "example": "1"
+            }
+          },
+          {
+            "name": "Cookie",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "access_token={{ACCESS_TOKEN}};_csrf={{CSRF_COOKIE}}"
+            }
+          },
+          {
+            "name": "csrf-token",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "{{CSRF_TOKEN}}"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      },
+      "delete": {
+        "tags": ["envTypeConfig"],
+        "summary": "Soft Delete envTypeConfig",
+        "description": "Soft Delete envTypeConfig",
+        "operationId": "softDeleteEnvtypeconfig",
+        "parameters": [
+          {
+            "name": "Cookie",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "access_token={{ACCESS_TOKEN}};_csrf={{CSRF_COOKIE}}"
+            }
+          },
+          {
+            "name": "csrf-token",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "{{CSRF_TOKEN}}"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      },
+      "patch": {
+        "tags": ["envTypeConfig"],
+        "summary": "Update envTypeConfig",
+        "description": "Update envTypeConfig",
+        "operationId": "updateEnvtypeconfig",
+        "parameters": [
+          {
+            "name": "Cookie",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "access_token={{ACCESS_TOKEN}};_csrf={{CSRF_COOKIE}}"
+            }
+          },
+          {
+            "name": "csrf-token",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "{{CSRF_TOKEN}}"
+            }
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "text/plain": {
```

Comment:
Do we want to do it this way and not with the json schema like you do for the other APIs?